nengo_bones/tests/test_format_notebook.py

```diff
@@ -4,10 +4,10 @@
 import re
 import sys
 
-from click.testing import CliRunner
-from nbconvert.preprocessors import ExecutePreprocessor
 import nbformat
 import pytest
+from click.testing import CliRunner
+from nbconvert.preprocessors import ExecutePreprocessor
```

Comment:
So all `from` statements have to come after `import` statements in a block? This is not how we do things right now. Is there a profile/setting that would be more similar to what we do now (both to reduce changes, and because I like what we do now)?

Reply:
There are configuration options, yes, I would be surprised if that wasn't configurable: https://pycqa.github.io/isort/docs/configuration/options/

I personally don't have strong opinions on what to do, as I would rather leave it up to `isort`. I would imagine that, over time, people would standardize on whatever the default option is, so I didn't spend any time looking through the configuration options to make it look like what we have now. The size of the diff is not a major factor as I imagine we would have all the changes in one commit that runs isort on anything anyhow. So yeah, I feel like we would need a pretty strong reason to opt for a particular configuration over the default.